src/main/resources/static/ws-test.html

```
@@ -1,117 +1,194 @@
+    <button id="addSubscriptionBtn" class="btn btn-secondary">ADD</button>
+  </div>
+
+  <h2>SEND MESSAGE</h2>
+  <div class="mb-3">
+    <label for="destinationPath" class="form-label">DESTINATION PATH:</label>
+    <input type="text" id="destinationPath" class="form-control" placeholder="/pub/send/message"/>
+  </div>
+  <div class="mb-3">
+    <label for="message" class="form-label">MESSAGE (JSON):</label>
+    <textarea id="message" class="form-control"
+              placeholder='{"targetUsername": "user", "message": "Hello", "sender": "admin"}'></textarea>
```

HTMLHint (1.5.0)

[error] The value of attribute [ placeholder ] must be in double quotes.

(attr-value-double-quotes)